integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -24,20 +37,30 @@ export class RasaNlu extends NluPlugin<RasaNluConfig> {
     return {
       serverUrl: 'http://localhost:5005',
       serverPath: '/model/parse',
+      alternativeIntents: { maxAlternatives: 15, confidenceCutoff: 0.0 },
     };
   }
 
-  async process(handleRequest: HandleRequest, jovo: Jovo): Promise<NluData | undefined> {
+  async process(handleRequest: HandleRequest, jovo: Jovo): Promise<RasaNluData | undefined> {
     const text = jovo.$request.getRawText();
     if (!text) return;
     try {
       const rasaResponse = await this.sendTextToRasaServer(text || '');
-      return rasaResponse?.data?.intent?.name
-        ? { intent: { name: rasaResponse.data.intent.name } }
-        : undefined;
+      if (rasaResponse.data.intent.name) {
+        return {
+          intent: {
+            name: rasaResponse.data.intent.name,
+            confidence: rasaResponse.data.intent.confidence,
+          },
+          alternativeIntents: this.mapAlternativeIntents(rasaResponse.data.intent_ranking),
+          entities: rasaResponse.data.entities.reduce(RasaNlu.mapEntities, {}),
+        };
+      } else {
```

Comment:
Code can be shortened by removing `else`. If the intent-name exists, `return` is called anyways and therefore the `else`-block can be omitted.

Reply:
i was wondering if this check is necessary or sensible too. Removed